modules/porous_flow/doc/content/modules/porous_flow/ates.md

```diff
@@ -0,0 +1,86 @@
+
+The purpose of this page is to describe a MOOSE model of an ATES system, with the goal of predicting $R$, and present a few results.  For significantly more background discussion, discussion of the modelling approach, and results, the reader is referred to [!citet](sheldon2021).
+
+## Model setup
+
+A single-well ATES system with radial symmetry is explored here, in which hot water is injected and produced from the same well.  Hence, this may be simulated using "RZ" coordinates:
+
+!listing modules/porous_flow/examples/ates/ates.i block=Problem
+
+!listing modules/porous_flow/examples/ates/ates.i block=GlobalParams
+
+The geometry is shown in [ates_geometry_fig].
+
+!media porous_flow/ATES_geometry.png caption=Geometry on a radial slice of the ATES model.  id=ates_geometry_fig
+
+In this example, the mesh is created by using a combination of MOOSE MeshGenerators, however, for accurate prediction of $R$, readers are encouraged to use a more sophisticated approach, such as the one outlined in [!citet](sheldon2021).
+
+!listing modules/porous_flow/examples/ates/ates.i block=Mesh
+
+The core physics is handled by the [PorousFlowFullySaturated](PorousFlowFullySaturated.md) Action, and [Kuzmin-Turek stabilization](kt.md) is used to minimise numerical diffusion of the temperature front which impacts computation of $R$:
+
+!listing modules/porous_flow/examples/ates/ates.i block=PorousFlowFullySaturated
+
+The porepressure and temperature are fixed at the extremities of the model:
+
+!listing modules/porous_flow/examples/ates/ates.i start=[outer_boundary_porepressure] end=[inject_heat1]
+
+The injection and production of heat from the borehole are the most complicated aspect of this input file.  The boundary conditions corresonding to injection are fixed temperature and constant rate of fluid flux:
+
+!listing modules/porous_flow/examples/ates/ates.i start=[inject_heat1] end=[produce_heat1]
+
+These boundary conditions are not always active.  They are controlled using
+
+!listing modules/porous_flow/examples/ates/ates.i start=[inject_on] end=[produce_on]
+
+with the `conditional_function` being:
+
+!listing modules/porous_flow/examples/ates/ates.i start=[inject] end=[produce]
+
+The boundary conditions corresponding to production are a withdrawal of fluid mass using a [PorousFlowSink](sinks.md) along with its corresponding withdrawal of heat energy (with the `use_enthalpy` flag set to true):
+
+!listing modules/porous_flow/examples/ates/ates.i start=[produce_heat1] end=[Controls]
+
+These are controlled using similar `Controls` as the injection phase.  Notice the `save_in` for the heat withdrawal.  This records the rate of heat leaving each node.  To find the total rate of heat loss (with units J.day$^{-1}$), a `NodalSum` is used:
```

Comment:
I can never remember if we can link to framework docs like `NodalSum.md`?

Reply:
Yea, seems like we can.  I think cross-module stuff may have been impossible at some stage, but framework stuff appears to work fine.